src/components/moss/MossLinkView.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import AppHeader from "../layout/AppHeader";
+
+const MossLinkView = (props) => {
+  return (
+    <div>
+
+      <div className="moss-container" style={{width: "30%", textAlign: "center"}}>
+        <h3>MOSS Results</h3>
+        <div style={{ height: 24 }}></div>
```

Comment:
would this work on mobile devices? you can test it using developer tools

Reply:
Fixed it along with any major responsiveness problem I could find